cmake/templates/BoostConfig.cmake.in

```diff
@@ -14,6 +14,18 @@ else()
   endfunction()
 endif()
 
+function(add_rt_library target_name)
```

Comment:
Please rename this function to avoid collisions with user's code (see http://cgold.readthedocs.io/en/latest/tutorials/control-structures.html#recommendation). I guess something like `hunter_boost_...`.